### ISSUE TITLE
[BUG]Possible file corruption

### ISSUE THREAD

**Issue by dubai3** [AnthonyMichaelTDM/DRG-Save-Editor#49]
Whenever I use the save editor to change season progress, it seems the file gets corrupted and resets almost all my progress to 0 upon booting the game. I have tried multiple times and disabling steam cloud saves as well. Still have old saves so nothing lost but maybe needs an update?

**Comment by AnthonyMichaelTDM** [comment]
Yes the editor needs an updateOn Jan 11, 2024, at 9:00 PM, dubai3 ***@***.***> wrote:﻿
Assigned #49 to @AnthonyMichaelTDM.

**Comment by AnthonyMichaelTDM** [comment]
Clarification though, does it only not work for season related things?

**Comment by KurrusIV** [comment]
> Clarification though, does it only not work for season related things?

I ran into this exact problem editing ONLY my blank cores, which aren't season related (to my knowledge). I changed the value from 1 to 400 and had the same issue. Haven't tested anything else.

**Comment by AnthonyMichaelTDM** [comment]
Then they might’ve changed the format/layout of the safe files, in which case the editor is going to need a lot of work to get working properly againSent from my iPhoneOn Feb 25, 2024, at 9:08 AM, KurrusIV ***@***.***> wrote:﻿

Clarification though, does it only not work for season related things?

I ran into this exact problem editing ONLY my blank cores, which aren't season related (to my knowledge). I changed the value from 1 to 400 and had the same issue. Haven't tested anything else.
